config_parser.go

```diff
@@ -22,6 +27,23 @@ func (r realConfigParser) parseConfig() config {
 	return cfg
 }
 
+func (r realConfigParser) expandTestsPathPattern(cfg config) []string {
```

Comment:
i suggest passing only what is required for the func (same applies for parseAdditionalParams)